ant/self-sign.properties

```diff
@@ -14,6 +14,7 @@ jks.alias=${build.socket.name}
 jks.keypass=!keypass
 jks.storepass=!storepass
 jks.passlength=10
+jks.host=0.0.0.0
```

Comment:
scratch my previous comment... I'm confusing it with `${jks.cn}`.  This should be good.